content/syllabuses/react-specialisation.md

```diff
@@ -5,13 +5,13 @@ _db_id: 30
 
 - {{< contentlink path="topics/web-frontend/react/intro-to-react" >}}
 - {{< contentlink path="topics/watch-this-space" >}}
-- {{< contentlink path="projects/tilde-mockups" flavour="javascript,react" >}}
+- {{< contentlink path="projects/tilde-mockups" flavour="react,javascript" >}}
```

Comment:
It should be just 'React' since React itself is built with JavaScript. No need to mention both 'React' and 'JavaScript' separately.